packages/docs-app/src/blueprint.md

```diff
@@ -30,71 +30,75 @@ The `main` module exports all symbols from all modules so you don't have to impo
 @### NPM installation
 
 1. Install the core package with an NPM client like `npm` or `yarn`, pulling in all relevant
-dependencies:
+   dependencies:
 
-  ```sh
-  yarn add @blueprintjs/core
-  ```
+
+```sh
+yarn add @blueprintjs/core
+```
 
 1. If you see `UNMET PEER DEPENDENCY` errors, you should manually install React:
 
-  ```sh
-  yarn add react react-dom react-transition-group
-  ```
+
+```sh
+yarn add react react-dom react-transition-group
+```
```

Comment:
revert this dedenting, it breaks up the list.

![image](https://user-images.githubusercontent.com/464822/35949266-7a548b80-0c25-11e8-8a7b-4ce6ecea475d.png)

actually looks like this whole file is pretty hosed. are you using some sort of automatic formatter? doesn't seem to be helping.